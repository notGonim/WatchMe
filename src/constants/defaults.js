export { default as Jumbotron } from '../components/jumbotron'
export { default as Footer } from '../components/footer'
export { default as According } from '../components/according'
export { default as Form } from '../components/form'
export { default as Header } from '../components/header'
export { default as Feature } from '../components/feature'

