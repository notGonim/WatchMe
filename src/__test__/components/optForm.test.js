import React from 'react';
import { render } from '@testing-library/react';
import { Form } from '../../constants/defaults';

describe('<OptForm />', () => {
    it('renders the <OptForm /> with populated data', () => {
        const { container, getByText, getByPlaceholderText } = render(
            <Form>
                <Form.Input placeholder="Email address" />
                <Form.Button>Try it now</Form.Button>
                <Form.Break />
                <Form.Text>Ready to watch? Enter your email to create or restart your membership.</Form.Text>
            </Form>
        );

        expect(getByText('Try it now')).toBeTruthy();
        expect(getByText('Ready to watch? Enter your email to create or restart your membership.')).toBeTruthy();
        expect(getByPlaceholderText('Email address')).toBeTruthy();
        expect(container.firstChild).toMatchSnapshot();
    
    
    
    })
})
