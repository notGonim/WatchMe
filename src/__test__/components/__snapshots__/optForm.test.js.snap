// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<OptForm /> renders the <OptForm /> with populated data 1`] = `
<div
  class="form__Container-z5rlus-0 gdoFde"
>
  <input
    class="form__Input-z5rlus-1 lluXXf"
    placeholder="Email address"
  />
  <button
    class="form__Button-z5rlus-2 hoeukk"
  >
    Try it now
     
    <img
      alt="try now"
      src="/images/icons/chevron-right.png"
    />
  </button>
  <div
    class="form__Break-z5rlus-4 bRzIrU"
  />
  <p
    class="form__Text-z5rlus-3 bGcGfr"
  >
    Ready to watch? Enter your email to create or restart your membership.
  </p>
</div>
`;
